index.js

```diff
@@ -3,134 +3,168 @@ const countries = require('./countries');
+          }
+        ]
+      },
+      {
+        name: 'origin',
+        label: 'Origin',
+        type: 'boolean',
+        htmlHelp: 'The origin point from which location from which the distance to a place result is calculated. <a href="https://developers.google.com/maps/documentation/javascript/places-autocomplete#add_autocomplete" target="_blank">More information here</a>',
+        choices: [
+          {
+            value: true,
+            showFields: [
+              'originLatitude', 'originLongitude'
+            ]
+          }
+        ]
+      },
+      {
+        name: 'originLatitude',
+        label: 'Origin Latitude',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'originLongitude',
+        label: 'Origin Longitude',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'bounds',
+        label: 'Bounds',
+        type: 'boolean',
+        htmlHelp: 'The area in which to search for places. <a href="https://developers.google.com/maps/documentation/javascript/places-autocomplete#add_autocomplete" target="_blank">More information here</a>',
+        choices: [
+          {
+            value: true,
+            showFields: [
+              'strictBounds', 'northEastLatitude', 'northEastLongitude', 'southWestLatitude', 'southWestLongitude'
+            ]
+          }
+        ]
+      },
+      {
+        name: 'northEastLatitude',
+        label: 'North East Latitude Bound',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'northEastLongitude',
+        label: 'North East Longitude Bound',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'southWestLatitude',
+        label: 'South West Latitude Bound',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'southWestLongitude',
+        label: 'South West Longitude Bound',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'strictBounds',
+        label: 'Strict Bounds',
+        type: 'boolean',
+        htmlHelp: 'Returns only those places that are strictly within the region defined by the given bounds. <a href="https://developers.google.com/maps/documentation/javascript/places-autocomplete#add_autocomplete" target="_blank">More information here</a>'
+      },
+      {
+        name: 'countries',
+        label: 'Countries',
+        type: 'array',
+        titleField: 'country',
+        limit: 5,
+        help: 'Restrict search to one or multiple countries (5 at max due to Google restriction). Might be not compatible with origin or bounds values.',
+        schema: [
+          {
+            name: 'country',
+            label: 'Country',
+            type: 'select',
+            choices: countries
+          }
+        ]
+      }, {
+        name: 'splitAddress',
+        label: 'Split Address',
+        type: 'checkboxes',
+        help: 'Optional: choose fields to split address to',
```

Comment:
Can `def` contain an array of all of the values, so that for the common case where they just want to split it to as many fields as it takes, we have them all by default?

Reply:
Good idea. I will add this (and update the version and the CHANGELOG file).

Reply:
I replaced "checkboxes" by "array". So no `def` possible now. But benefits for the user are:
- easily reorganize fields order
- ability to pass a custom label